docs/basics.md

```diff
@@ -132,10 +132,10 @@ For instance, `PathPrefix: /products` would match `/products` but also `/product
 Since the path is forwarded as-is, your backend is expected to listen on `/products`.
 
 Use a `*Strip` matcher if your backend listens on the root path (`/`) but should be routeable on a specific prefix.
-For instance, `PathPrefixStrip: /products` would match `/products` but also `/products/shoes` and `/products/shirts`.  
```

Comment:
2 spaces have a meaning in markdown.

> When you do want to insert a <br /> break tag using Markdown, you end a line with two or more spaces, then type return.
https://daringfireball.net/projects/markdown/syntax#p

Reply:
Oh I didn't know that. Then here are options:

- keep the two spaces at EOL
- replace the two spaces at EOL with `<br>`
- replace the two spaces at EOL with `\n\n` to separate paragraphs
- use different formatting, such as `!!! note` or lists 

I still don't like the significant trailing spaces since it's very easy to mess up. I think using `<br>` is better if `<br>` is really needed. Otherwise, the best option is to avoid `<br>` at all - use separate paragraphs or other formatting. 

Let me know - I'll make changes in this PR accordingly.

Reply:
> keep the two spaces at EOL